src/lib/y2storage/planned/md.rb

```diff
@@ -101,6 +101,24 @@ def add_devices(md_device, devices)
         end
       end
 
+      # Whether the given name matches the name of the planned MD
+      #
+      # Apart from directly comparing the strings, this method is also
+      # able to compare a string with a format like "/dev/md0" (by default,
+      # the planned MD uses the libstorage-ng format for MD names, which looks
+      # like "/dev/md/0").
+      #
+      # @param value [String] name been searched for
+      # @return [Boolean] true if the names match
+      def name?(value)
```

Comment:
In the hypothetical case that `libstorage-ng` returns `/dev/md0` and `autoyast` indicates `dev/md/0`, this method returns `false`. Should it return `true` for such situation? Or that situation cannot happen?

Reply:
It shouldn't happen right now, at least using the typical AutoYaST profile.

But to make sure, I will try to enforce that situation by defining an unusual (but valid) profile and simulating the "/dev/md0" behavior for libstorage-ng. Maybe there is an opportunity window for this to fail in the future.